src/Pages/Experts/Experts.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import Address from './Address';
+import Modal from './Component/Modal';
+import { DEFAULT_LAT, DEFAULT_LNG } from '../../config';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+export default function Experts() {
+  const { kakao } = window;
```

Comment:
```suggestion
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';

const { kakao } = window;

export default function Experts() {
```
이미 head 태그에서 스크립트를 가져오고 있기 때문에 바깥에서 구조분해 할당 해주어도 문제 없습니다

Reply:
네